util/src/main/java/tk/zielony/randomdata/util/StringIPAddressGenerator.java

```diff
@@ -9,38 +9,42 @@
 
 public class StringIPAddressGenerator extends Generator<String> {
 
-    public enum classRange {
-        A, B, C
+    public enum ClassRange {
+        A(1, 126), B(128, 191), C(192, 223); // check the values, I'm not sure if these are okay
+
+        int min,max;
```

Comment:
formatting

Reply:
How would you like this to be formatted? With new lines?